cmd/cmd.go

```diff
@@ -46,6 +46,7 @@ func New() *cobra.Command {
 		newCombineCmd(newCombineFunc),
 		newAlphaCmd(
 			newAddValidatorsCmd(runAddValidatorsSolo),
+			newViewClusterManifestCmd(runViewClusterManifest),
```

Comment:
fix indent

Reply:
wdym?

Reply:
Nvm, when viewing the PR, i couldn't see a tab space